### DISCUSSION TITLE
Error with : /bin/bash first_run.dev.sh command during setup. Also error with : docker-compose -f docker-compose.dev.yml --env-file .env up

### DISCUSSION THREAD

**Discussion by passeng02** [frg-fossee/eSim-Cloud#543] [Q&A]
**/bin/bash first_run.dev.sh** 
`ERROR: Service 'celery' failed to build : The command '/bin/sh -c apk add --no-cache mariadb-connector-c-dev ;    apk add --update alpine-sdk &&     apk add libffi-dev openssl-dev &&     apk --no-cache --update add build-base ;    apk add --no-cache --virtual .build-deps gcc python3-dev musl-dev mariadb-dev    && pip install --upgrade pip    && pip install --trusted-host pypi.python.org -r requirements.txt     && apk del .build-deps' returned a non-zero code: 1`

**docker-compose -f docker-compose.dev.yml --env-file .env up:** 
`ERROR: failed to solve: process "/bin/sh -c apk add --no-cache mariadb-connector-c-dev ;    apk add --update alpine-sdk &&     apk add libffi-dev openssl-dev &&     apk --no-cache --update add build-base ;    apk add --no-cache --virtual .build-deps gcc python3-dev musl-dev mariadb-dev    && pip install --upgrade pip    && pip install --trusted-host pypi.python.org -r requirements.txt     && apk del .build-deps" did not complete successfully: exit code: 1`

ERROR: Service 'celery' failed to build : Build failed

**Comment by firuza** [comment]
This issue has been fixed in the develop branch. Please use it as develop branch is the most recent and updated one.

**Reply by passeng02** [reply]
My forked repo did not have develop branch up to date with fossee repo - develop branch. Syncing the branch worked. Thanks!
